rasgotransforms/rasgotransforms/dtypes.py

```diff
@@ -0,0 +1,40 @@
+DTYPES = {
```

Comment:
We have other places in our code base where we map actual data types to simplified "rasgo" data types.  Is there any chance we could benefit from a similar treatment here?

Reply:
Yeah I think a mapping will make this easier going forward.  We do have a mapping to numeric, categorical, or datetime in the API, but we don't use it for storing column metadata; we store ansi sql types in that table so I did this to avoid making a change. I think we can push this for now and then change the mapping after we think about types some more because there have been a few points making it worth a discussion that have come up this week.